core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.html

```diff
@@ -115,6 +108,15 @@
     bottom: 13px;
   }
 
+  topics-list .delete-icon {
```

Comment:
Don't change the position of this line of code i.e, it looks like an unwanted change.

Reply:
Done